### PR TITLE
mwpw-153237: Merch @ Scale Test Content

### PR DESCRIPTION
**Pull request by Axelcureno**
Merch at Scale static content to be used for performance checks, etc. 
For now only including A.com merch cards. Adobe Home and CCD cards coming soon.
Supports milolibs configuration as URL param.

Jira ticket: https://jira.corp.adobe.com/browse/MWPW-153237

Test URLs:
- Before: https://main--mas--adobecom.hlx.live/web-components/test/static/merch-card/index.html
- After: https://mwpw-153237--mas--adobecom.hlx.live/web-components/test/static/merch-card/index.html

### CODE REVIEW

**Comment by yesil** [comment]
https://mwpw-153237--mas--adobecom.hlx.live/web-components/test/static/merch-card/segment.html

I noticed a CLS at https://pagespeed.web.dev/analysis/https-mwpw-153237--mas--adobecom-hlx-live-web-components-test-static-merch-card-segment-html/b56tgbpjw1?form_factor=mobile

If we use those pages for PSI checks we need to clear the CLS issues.

File: web-components/src/global.css.js [hunk 1]
```diff
@@ -132,15 +132,6 @@ styles.innerHTML = `
 
     --ellipsis-icon: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><circle cx="2" cy="2" r="2" fill="%232c2c2c" transform="translate(6 6)"/><circle cx="2" cy="2" r="2" fill="%232c2c2c" data-name="Ellipse 71" transform="translate(12 6)"/><circle cx="2" cy="2" r="2" fill="%232c2c2c" transform="translate(0 6)"/></svg>');
 
-    /* callout */
```
Comment (3ch023) [inline web-components/src/global.css.js:135]: why are the callout changes being removed?

Comment (Axelcureno) [reply web-components/src/global.css.js]: Thanks for catching this, I have reverted those changes made by mistake. 🫠

**Comment by Axelcureno** [comment]
> I noticed a CLS at https://mwpw-153237--mas--adobecom.hlx.live/web-components/test/static/merch-card/segment.html
> 
> https://pagespeed.web.dev/analysis/https-mwpw-153237--mas--adobecom-hlx-live-web-components-test-static-merch-card-segment-html/b56tgbpjw1?form_factor=mobile
> 
> If we use those pages for PSI checks we need to clear the CLS issues.

@yesil thanks, I have fixed it: https://pagespeed.web.dev/analysis/https-mwpw-153237--mas--adobecom-hlx-live-web-components-test-static-merch-card-segment-html/118c3f45vd?form_factor=desktop
Screenshot 2024-07-11 at 6 40 51 PM

**Review by npeltier — CHANGES_REQUESTED** [review]
i would wait for move to milo :) and also i would separate consumer test from the code

**Comment by Axelcureno** [comment]
Moved to https://github.com/adobecom/milo/pull/2677

**Pull request closed without merging**